docs/UserGuide.md

```diff
@@ -79,18 +79,26 @@ Format: `add n/STUDENT_NAME c/CONTACT_NUMBER a/HOME_ADDRESS [r/RISK_LEVEL]`
 
 **Parameters**:
 1. Name
-   - Alphabetical characters only
+   - Cannot contain symbols, alphabetical characters only
+   - Must be unique
+   - Maximum of 100 characters
 2. Contact Number
    - Numbers only, must be 8 characters long
 3. Home Address
-   - Maximum of 200 characters
+   - Maximum of 200 characters, cannot be blank
 4. Risk Level
    - Must be `high`, `medium`, or `low`
 
 Examples:
 * `add n/John c/81349705 a/Yishun Street 56 Blk 21 #05-07`
 * `add n/Sally c/94149785 a/Woodlands Street 11 Blk 888 #08-08 r/low`
 
+Invalid examples:
+* `add n/Sally c/94149785` (missing address)
+* `add n/Sa11y! c/1234 a/Woodlands Street 11 Blk 888 #08-08` (invalid name and phone number)
+* `add n/Sa11y! c/94149785 a/` (address is blank)
```

Comment:
Maybe `Sally` instead of `Sa11y` is better here because the error is address only.